stage1_fly/run/main.go

```diff
@@ -218,8 +218,16 @@ func stage1() int {
 		return 1
 	}
 
-	// TODO: insert environment from manifest
-	env := []string{"PATH=/bin:/sbin:/usr/bin:/usr/local/bin"}
+	defaultEnv := []string{"PATH=/bin:/sbin:/usr/bin:/usr/local/bin"}
```

Comment:
@steveeJ do you know why this has `/sbin` but not `/usr/sbin` and `usr/local/bin`? This seems inconsistent with the other paths.